dev.html

```diff
@@ -5,8 +5,9 @@
     <!-- Content-Security-Policy connect-src:
       - https://*.mapbox.com for COVID-19 Dashboard maps
       - https://opendata.datacommons.io to fetch COVID-19 Dashboard data
+      - https://www.google-analytics.com for the dashboard map
```

Comment:
can you please check if it works without adding `https://www.google-analytics.com`? since apparently the issue was something else. and if we do need it, can you update the comment in both files with a bit more detail - `for COVID-19 Dashboard maps in Safari` or something

Reply:
I will try doing that